wordpress-mobile/WordPress-iOS
# [Jetpack] Add Jetpack banner to Themes

## Description

This PR adds the "Jetpack powered" banner to the Themes view.

## Testing

1. Open the WordPress app
2. In the My Site view, select a site that is either hosted at WordPress.com or has Jetpack connected
3. Tap "Themes"
4. Observe the Jetpack banner

### Items to test:
- Large accessibility sizes
- Dark mode
- Landscape orientation
- iPad
- Banner should not appear in Jetpack app
- Banner should not appear when the feature flag is disabled
- Themes view should not appear for self-hosted sites that are not Jetpack connected
- When scrolling the banner should appear and disappear with a smooth animation

| Reader banner iPhone | Reader banner iPad |
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-07-19 at 12 16 13](https://user-images.githubusercontent.com/2092798/179801427-fd4462d7-d363-497a-9cfa-32afee8d70b5.png) | ![Simulator Screen Shot - iPad (9th generation) - 2022-07-19 at 12 18 59](https://user-images.githubusercontent.com/2092798/179801460-3da34300-d5fe-4508-9427-8d95c654e98e.png) |

## Regression Notes
1. Potential unintended areas of impact
    - Themes view functionality (e.g. searching, selecting a theme)

2. What I did to test those areas of impact (or what existing automated tests I relied on)
    - Manual testing

3. What automated tests I added (or what prevented me from doing so)
    - None

PR submission checklist:
